test_opensearchpy/test_async/test_server/test_helpers/conftest.py

```diff
@@ -43,12 +43,30 @@ async def client() -> Any:
 
 @fixture(scope="function")  # type: ignore
 async def opensearch_version(client: Any) -> Any:
-    info = await client.info()
+    """
+    yields the version of the OpenSearch cluster
+    :param client:
+    :return:
+    """
+    info = client.info()
```

Comment:
This is missing an `await`, so you get the error.

Reply:
that was also a dumb copy/paste error if you look at the wider scope. fixing. thanks, @dblock.